docs/beyond_basics/101-173-contributing.rst

```diff
@@ -0,0 +1,28 @@
+We would delighted to hear from you in any form.
+
+.. figure:: ../artwork/src/join.svg
+   :width: 50%
+
+The following resources could be helpful:
+
+Contributing to the handbook
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+* Take a look at the section :ref:`contribute` for more information.
+
+Contributing to DataLad
+^^^^^^^^^^^^^^^^^^^^^^^
+
+* **Show your support**: If you like DataLad you can show your support in various meaningful ways. You can `"star" <https://github.com/datalad/datalad/stargazers>`_ the project on GitHub. You can follow the `DataLad Twitter Account <https://twitter.com/datalad>`_ and like and retweet our updates. You can cite the `paper about DataLad <https://joss.theoj.org/papers/10.21105/joss.03262>`_ in academic works or blog posts. You can tell your friends, family, and colleagues about it, or come chat with us about what you do on `Matrix <https://app.element.io/#/room/%23datalad:matrix.org>`_ (a free, decentral, and secure communication network). Whatever you do - thank you!
+
+* **Contributing on GitHub**: A most valuable contribution is your time. We are interested and grateful for opinions, bug reports, feature requests, patches, larger code contributions, or simply a notice what you use DataLad for. Find the relevant repository, be that `github.com/datalad/datalad <https://github.com/datalad/datalad>`_ (the main repository), `github.com/datalad-datasets <https://github.com/datalad-datasets>`_ (many open datasets), or any :term:`DataLad extension`, and open issues or pull requests. DataLad's `CONTRIBUTING <https://github.com/datalad/datalad/blob/master/CONTRIBUTING.md>`_ file has tons of technical and social information to get you started with code contributions. But don't be intimidated by the wealth of information you will find in there. We'll be happy to help you at any stage. Also, you can take a look at technical docs (`docs.datalad.org <http://docs.datalad.org/>`_) and in particular the :ref:`Design documents <design>` that shed light on the internal design principles of the software.
+
+* **Write an extension!** If you have unique use cases, you can write your own :term:`DataLad extension` for it, that can provide any number of additional DataLad commands that are automatically included in DataLad's command line and Python API. Our `extension template <https://github.com/datalad/datalad-extension-template>`_ is the best starting point. It contains an example command implementation, and will have test setup and packaging configurations in place already. If you want to, you can register your extension against DataLad's extension registry at `github.com/datalad/datalad-extensions <https://github.com/datalad/datalad-extensions>`_ -- if your project is included, we can continuously check whether current versions of DataLad work with your extension.
```

Comment:
We could also may be say that we could include their contributed videos to our YouTube channel? e.g. if someone makes a nice tutorial etc

Reply:
Another type of contribution, which is already implicit in the ones you've covered, is contributions to the docs. Specifically when it comes to increasing the clarity of instructions, descriptions, etc. I think it could be useful to explicitly note (whether under its own heading or incorporated into an existing section) that we value contributions where people help us to make DataLad more understandable and user-friendly. E.g. "We want to improve the clarity of the DataLad documentation. If you tried to implement DataLad in a specific way and the existing documentation didn't make sense, or wasn't clear enough or even confusing, please help us fix it. Let us know (by creating an issue or sending a message on Matrix) that the instructions could have been clearer, or that it didn't cover your use case, or led you along the wrong path. And if you have suggestions for improvements, let's incorporate them!"

Reply:
done! And thanks for the great wording!